examples/src/main/python/mllib/naive_bayes_example.py

```diff
@@ -50,8 +61,17 @@ def parseLine(line):
     # Make prediction and test accuracy.
     predictionAndLabel = test.map(lambda p: (model.predict(p.features), p.label))
     accuracy = 1.0 * predictionAndLabel.filter(lambda (x, v): x == v).count() / test.count()
+    print('\n\tmodel accuracy %.4f\n' % accuracy)
 
     # Save and load model
-    model.save(sc, "target/tmp/myNaiveBayesModel")
-    sameModel = NaiveBayesModel.load(sc, "target/tmp/myNaiveBayesModel")
+    output_dir = '/tmp/myNaiveBayesModel'
```

Comment:
This doesn't work because `/tmp` is not always available, which is why we used `target/tmp`. Let's just assume that users run this example code under the `spark/` folder, or we can document it in the usage.